Storage/src/Connection/Rest.php

```diff
@@ -600,4 +602,28 @@ protected function supportsBuiltinCrc32c()
     {
         return Builtin::supports(CRC32::CASTAGNOLI);
     }
+
+    /**
+     * Add the required retry function and send the request.
+     *
+     * @param string $resource resource name, eg: buckets.
+     * @param string $method method name, eg: get
+     * @param array $args
+     */
+    private function sendWithRetry($resource, $method, $args)
+    {
+        $retryMap = [
+            'projects.resources.serviceAccount' => 'serviceaccount',
+            'projects.resources.hmacKeys' => 'hmacKey',
+            'bucketAccessControls' => 'bucket_acl',
+            'defaultObjectAccessControls' => 'default_object_acl',
+            'objectAccessControls' => 'object_acl'
+        ];
+        if (array_key_exists($resource, $retryMap)) {
+            $args['restRetryFunction'] = $this->getRestRetryFunction($retryMap[$resource], $method, $args);
+        } else {
+            $args['restRetryFunction'] = $this->getRestRetryFunction($resource, $method, $args);
+        }
```

Comment:
```suggestion
        if (array_key_exists($resource, $retryMap)) {
            $resource = $retryMap[$resource];
        }
        $args['restRetryFunction'] = $this->getRestRetryFunction($resource, $method, $args);
```

Reply:
Done (in a slightly different way since the $resource is needed intact down the line)